test/unit/browser/invalidation_mixin_test.dart

```diff
@@ -36,9 +36,9 @@ void main() {
         expect(thing.invalid, isTrue);
 
         // ignore: unawaited_futures
-        onValidation.then(expectAsync((ValidationStatus status) {
+        onValidation.then(expectAsync1((ValidationStatus status) {
           expect(status, equals(ValidationStatus.cancelled));
-        }, count: 1));
```

Comment:
I like being explicit with the counts when the count is largely what is being tested, but if people don't like it then that's fine.